### PR TITLE
feat: added outgoing split transfer status page

### PR DESCRIPTION
**Pull request by justinenerio**
## Changes

- Added Outgoing Split transfer status pages
- Added Timeline widget

## Screenshots/Videos
status

Canceled Flow

https://user-images.githubusercontent.com/28533130/192702615-607d051c-875e-4df8-9f83-762371ab7cca.mp4

Success Flow

https://user-images.githubusercontent.com/28533130/192702696-8f9e930a-7f67-4320-9a00-f31affee7be2.mov

## Checklist

- [x] PR is ready for review (if not, it should be a draft).
- [x] PR title follows [Conventional Commits][1] guidelines.
- [x] Screenshots/video added.
- [ ] Tests added.
- [x] Self-review done.

[1]: https://www.conventionalcommits.org/en/v1.0.0/

### CODE REVIEW

**Comment by antoineherzog** [comment]
@justinenerio A minor feedback on the font-size, it is a semi-bold. It looks bold here

**Comment by justinenerio** [comment]
@antoineherzog updated code and screenshots. font weight should now be reduced.

File: packages/cryptoplease/pubspec.yaml [hunk 1]
```diff
@@ -60,6 +60,7 @@ dependencies:
   solana:
     path: ../solana/
   sqlite3_flutter_libs: ^0.5.10
+  timelines: ^0.1.0
```
Comment (ookami-kb) [inline packages/cryptoplease/pubspec.yaml:63]: We try to minimize the number of dependencies, especially when it comes to widgets. This should be pretty easy to implement on our own, so let's rather create a component in `cryptoplease_ui`.

It's even better to make it a separate PR.

**Review by ookami-kb — CHANGES_REQUESTED** [review]

File: packages/cryptoplease/pubspec.yaml (hunk 1 above)
Comment (justinenerio) [reply packages/cryptoplease/pubspec.yaml:63]: Understood. Have created a separate PR for this widget. 

Will update this PR once other one is merged.

Comment (justinenerio) [reply packages/cryptoplease/pubspec.yaml]: Removed `timelines` dependency and replaced it with custom `CpTimeline`.

**Review by ookami-kb — APPROVED** [review]
